ai-assistants/utils/blobConversion.ts

```diff
@@ -0,0 +1,58 @@
+import { logger } from "deco/observability/otel/config.ts";
+import { AssistantIds } from "../types.ts";
+
+export default function base64ToBlob(
+  base64: string | ArrayBuffer | null,
+  context: string,
+  assistantIds?: AssistantIds,
+): Blob {
+  let regex =
+    /^data:(audio\/[a-z]+|video\/[a-z]+|audio\/mp[34]|video\/mp4);base64,(.*)$/;
```

Comment:
`audio/mp3` isn't used (it is `audio/mpeg`).

you want to match with any audio/video? if so:

```suggestion
    /^data:(audio\/[a-z0-9\-\+\.]+|video\/[a-z0-9\-\+\.]+);base64,(.*)$/;
```